reference-mod/.gitignore

```diff
@@ -111,8 +112,11 @@ gradle-app.setting
 
 **/build/
 
-# Common working directory
+# Common working directories
 run/
+runClientPrimary/
```

Comment:
Again, loom 1.1 and IntelliJ 2023 supports using the same folder for client and server

Reply:
Are you sure? I had a project where ClothConfig was complaining bcuz of that, even tho loom 1.1 was used there. 

*(I'm still using the old IntelliJ version, tho, so IDK)*

I will rewrite this soon ™

Reply:
We used the same folder for SkinShuffle and it worked fine :shrug: - might depend on what the dev does in their mod.

Reply:
Should I let it stay and put the information in there, that if they get the same problem, they could use this? Or rly just get rid of it...

Here was the conversation when i had this issue fyi...
https://discord.com/channels/507304429255393322/507982478276034570/1147464234414903326